setup.py

```diff
@@ -37,6 +37,7 @@
         'pandas>=0.18',
         'deap',
         'efel>=2.13',
+        'arbor>=0.7',
```

Comment:
Could we make this an extra, I'd prefer not to depend on arbor for now, since it will be not be the general use case for bluepyopt

Reply:
Sure, did that in de52dea.